### ISSUE TITLE
boolean facet issue

### ISSUE THREAD

**Issue by ballabusta** [elastic/search-ui#556]
Hi - would it make sense to differentiate between the facet value name and the label of the facet ?
For BooleanFacet component, it's currently the same, say you have the 'In stock' boolean facet, you might want to have a label named 'Status'.
Thank you!

**Comment by JasonStoltz** [comment]
Good point. I think the `Facet` field has a `label` property, right? So you should be able to customize the facet label.

**Comment by ballabusta** [comment]
The problem is that the _label_ is also the assigned to the facet name, so there is no way to differentiate.

https://github.com/elastic/search-ui/blob/master/packages/react-search-ui-views/src/BooleanFacet.js#L38

I wonder if the facet name can be added as a new prop.

**Comment by JasonStoltz** [comment]
Ah, I see. That would be better. In the meantime, I would try creating a Custom View.
